packages/react/src/components/Modal/Modal.js

```diff
@@ -12,6 +12,7 @@ import Button from '../Button';
 import { settings } from 'carbon-components';
 import { Close20 } from '@carbon/icons-react';
 import FocusTrap from 'focus-trap-react';
+import '../../tools/toggle-class';
```

Comment:
Is it possible for this polyfill to conflict with a user-supplied one in the same codebase? Noticed that there are a couple out there on NPM and was curious how this would work alongside if one of those are already being provided

Reply:
We can use one in NPM, too. Wrt conflict - There is a logic that detects pre-existing native/polyfill version.

Reply:
Another approach is making it a simple function rather than a polyfill, like:

```javascript
function toggleClass(element, name, add) {
  if (element.classList.contains(name) === !add) {
    element.classList[add ? 'add' : 'remove'](name);
  }
}
```

Reply:
Yeah, was thinking it might be easier to just implement the logic with supported APIs instead of trying to use the second arg.